cockatrice/src/window_main.cpp

```diff
@@ -795,3 +799,8 @@ void MainWindow::refreshShortcuts()
     aExit->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aExit"));
     aCheckCardUpdates->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aCheckCardUpdates"));
 }
+
+void MainWindow::notifyUserAboutUpdate()
+{
+    QMessageBox::information(this, tr("Information"), tr("A new version of the cockatrice client is available for download."));
```

Comment:
I think if the client itself is giving the url it's OK.

We can add an entry in the help menu to link to download page. We can also put a button on the dialog to go there.

I'm going to make a new repo under the org for weekly builds. We can change the build scripts to automatically publish releases to the github repo, and to upload and attach the builds. Then we'll have a centralized download page. Full releases can still happen on the cockatrice repo releases tab.